fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/ReprocessLoanTransactionsServiceImpl.java

```diff
@@ -208,8 +209,12 @@ private void handleChangedDetail(final ChangedTransactionDetail changedTransacti
 
             if (oldTransaction != null) {
                 loanAccountTransfersService.updateLoanTransaction(oldTransaction.getId(), newTransaction);
-                // Create reversal journal entries for old transaction if it exists (reverse-replay scenario)
-                loanJournalEntryPoster.postJournalEntriesForLoanTransaction(oldTransaction, false, false);
+                // Only create reversal journal entries for old transaction if the old transaction is not already
```

Comment:
Are you sure? I dont recall any reported bug whether something was incorrectly or double accounted.